CGPs/cgp-0128/mainnet.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+      "address": "0x0239b96D10a434a56CC9E09383077A0490cF9398",
+      "function": "deposit()",
+      "args": [],
+      "value": "10000000000000000000000000"
+    },
+    {
+      "address": "0xC668583dcbDc9ae6FA3CE46462758188adfdfC24",
+      "function": "approve(address,uint256)",
+      "args": [
+        "0x3d79EdAaBC0EaB6F08ED885C05Fc0B014290D95A",
+        "10000000000000000000000000"
+      ],
+      "value": "0"
+    },
+    {
+      "address": "0x3d79edaabc0eab6f08ed885c05fc0b014290d95a",
+      "function": "mint",
+      "args": [
```

Comment:
Did you try to dry-run this to see if tx revert? could be done in a fork.

Reply:
Yes I did

Reply:
This is howto:
https://clabsco.notion.site/Test-on-forked-celo-mainnet-d77e5541323245ca995af002dbbcfd68?pvs=74